docs: Promote GitHub Web UI guides (#62911)
gravitational/teleport: docs/pages/machine-workload-identity/deployment/github-actions.mdx

Environment variables are configured by this action and these automatically
configure `tsh` and `tctl` to use this identity.

<Admonition type="info" title="Guided integration available">

>     Why not use the GitHub Action + SSH integration guide via the

Reviewer: ```suggestion
    You can use the GitHub Action + SSH integration guide via the
```

Suggesting a more neutral tone

all the pods contained within the cluster, but this could just as easily be
modified to deploy to a Kubernetes cluster with `kubectl` or `helm`.

<Admonition type="info" title="Guided integration available">
    Why not use the GitHub Action + Kubernetes integration guide via the
    Teleport Web UI?
    
    The guide walks you through connecting your GitHub Actions
    workflow to Teleport and giving it the permissions it needs to access your
    enrolled Kubernetes clusters.

>     Find the guide in the Web UI;<br />

Reviewer: ```suggestion
    Find the guide in the Web UI:<br />
```
